index.html

```diff
@@ -266,158 +320,202 @@ <h2 class="h-text-white h-text-size">
                 </li>
               </ul>
             </div>
-            <div class="btn-group">
-              <button type="button" class="btn-2">See Project</button>
-            </div>
+            <!-- <div class="btn-group">
+            <button type="button" class="btn-2">See Project</button>
+          </div> -->
           </div>
         </div>
       </div>
     </section>
 
-    <section class="container bg-color-1">
-      <div class="bg-image-2">
-        <section class="about">
-          <h2 class="h-text-lg">About me</h2>
-          <p class="headline-inside3">
-            Hello I’m a software developer! I can help you build a product ,
+    <!--About me-->
+    <section class="about" id="about">
+      <div class="about-section">
+        <div class="about-sub-section">
+          <h1 class="about-header header">About me</h1>
+          <p class="about-description">
+            Hello I'm a software developer! I can help you build a product ,
             feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
+            you like what you see and have a project you need coded, don't
```

Comment:
- [ ] The background image of the `About` section in the desktop version is missing. It would be great to add this so as to follow the Figma design requirements.
- Your design:
![Annotation 2022-02-26 053859](https://user-images.githubusercontent.com/85113567/155829901-b221b910-5e88-4a9d-8f66-d816b3f75775.jpg)
- Figma design:
![Annotation 2022-02-26 053841](https://user-images.githubusercontent.com/85113567/155829904-dd553aab-499b-456b-b730-f960e84cef1a.jpg)

Reply:
Thank you, I did that.

Reply:
- [ ] [OPTIONAL] thanks for making that change. you can also change the background color of this section as it is supposed to be slight lighter than pure black.

Reply:
Thank you, I did that